packages/xo-server-auth-saml/src/index.js

```diff
@@ -20,6 +20,7 @@ export const configurationSchema = {
       title: 'Certificate',
       description: "Copy/paste the identity provider's certificate",
       type: 'string',
+      multiline: true,
```

Comment:
I think it's a good idea to add namespace this entry because it's not valid JSON schema, for instance by prefixing with `$`.